build.sbt

```diff
@@ -137,6 +137,8 @@ lazy val controlApi = (project in file("target/control/java"))
       "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonVersion,
       "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
       "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % jacksonVersion,
+      "org.apache.httpcomponents" % "httpclient" % orgApacheHttpVersion,
```

Comment:
Feels like a big dependency just to use to use the `UrlEncoderUtils`... or were you planning to eventually refactor the current uses of `HttpURLConnection` with `HttpClient` instead? Does Armeria have any built in classes for Url Encoding?

Reply:
Even when the library template is set to `native`, OpenAPI Generator seems to generate an API that depends on Apache HttpClient when handling `x-www-form-urlencoded` requests/responses:

 - https://openapi-generator.tech/docs/generators/java/
 - https://gist.github.com/ognis1205/526371ddaed10cb2e96ab83ccc24012b

Could you clarify if you are referring to the `cli` build? If so, I completely agree with your point, and I will make the necessary adjustments.

Reply:
I have removed the Apache HttpClient dependency from the `cli` project. It would be great if you could share your thoughts on the `controlApi` project whenever you have some time.